### ISSUE TITLE
[oracle] SHOW unparsable

### ISSUE THREAD

**Issue by osarier** [sqlfluff/sqlfluff#8102]
### Search before asking

- [x] I searched the [issues](https://github.com/sqlfluff/sqlfluff/issues) and found no similar issues.

### What Happened

Linting the example below leads to a 'Found unparsable section' error.

### Expected Behaviour

The example should be parsed correctly. Show doesnt seem to be supported at all currently
See: https://docs.oracle.com/en/database/oracle/oracle-database/26/sqpug/SHOW.html

### Observed Behaviour

Linting the example below gives the following error:
`Line 1, Position 1: Found unparsable section: 'show errors'`

SHOW is currently not supported at all.

### How to reproduce

Lint the following example:

```sql
show errors
``` 
Other options don't work as well.

### Dialect

oracle

### Version

4.2.1
python 3.11.3

### Configuration

[sqlfluff]
dialect = oracle

### Are you willing to work on and submit a PR to address the issue?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/sqlfluff/sqlfluff/blob/main/CODE_OF_CONDUCT.md)

**Comment by Purva-225** [comment]
Hi @osarier, I'd like to work on this issue. If it's still available, could you please assign it to me? Thank you!
